cdm/core/src/main/java/ucar/nc2/filter/Classifier.java

```diff
@@ -2,20 +2,26 @@
 
 import java.io.IOException;
 import ucar.ma2.Array;
-import ucar.ma2.DataType;
 import ucar.ma2.IndexIterator;
+import ucar.nc2.NetcdfFile;
+import ucar.nc2.Variable;
+import ucar.nc2.dataset.NetcdfDatasets;
```

Comment:
check for unused imports, I think `NetcdfDatasets`?

Reply:
Deleted the ones not used